.storybook/webpack.config.js

```diff
@@ -1,12 +1,30 @@
 const merge = require("lodash/merge");
 const path = require("path");
 const wixStorybookConfig = require("yoshi/config/webpack.config.storybook");
+const { StylableWebpackPlugin } = require("@Stylable/webpack-plugin");
```

Comment:
```suggestion
const { StylableWebpackPlugin } = require("@stylable/webpack-plugin");
```

i believe this is a typo (capital S), noticed on clean install